core/src/main/scalajvm/sttp/tapir/internal/FileChunk.scala

```diff
@@ -0,0 +1,16 @@
+package sttp.tapir.internal
+
+import sttp.tapir.RangeValue
+
+import java.io.RandomAccessFile
+
+object FileChunk {
+
+  def prepare(tapirFile: TapirFile, range: RangeValue): Array[Byte] = {
```

Comment:
maybe we could return an `InputStream` here? So that we don't have to read the whole chunk into memory